flutter_appauth_platform_interface/lib/src/errors.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/services.dart';
+
+/// The details of an error thrown from the underlying
+/// platform AppAuth libraries.
```

Comment:
IMO the sentence doesn't make sense and/or reads awkward, particularly around "underlying platform AppAuth libraries". "Underlying platform" results in a single platform (Android or iOS) so there should only be one library. 

Something along the lines of


```
/// The details of an error thrown from the underlying
/// platform's AppAuth SDK
```

would IMO make more sense. It also uses "SDK" that aligns with how AppAuth for Android and iOS refers to themselves as SDKs so brings consistency there

Reply:
On a related note, if you agree with this, could you update other similar API docs in a similar way?